src/app/reddit/post-subtitle/post-subtitle.component.html

```diff
@@ -11,7 +11,7 @@
 <br />
 <span
   ><a
-    *ngIf="post.url && post.url.search('www.reddit.com') == -1"
+    *ngIf="post.url && post.url.indexOf('www.reddit.com') === -1"
```

Comment:
To be clear, this would never be an issue, as I am _avoiding_ anything that contains the URL. However, I don't like marking anything as false positive...